### PR TITLE
Add SimpleMDE and Fix Image Paste for Issue/Comment Editor

### PR DESCRIPTION
**Pull request by blueworrybear**
This PR update #9132 and #8834. Since the paste event should bind with SimpleMDE, I decide to update two issues in this PR.

This PR does not implement user option to enable/disable SimpleMDE. I'm not sure if it's a good idea to enable this feature with this PR because it requires to add a new column into database.  Need some discussions. 🤔

### CODE REVIEW

**Comment by lunny** [comment]
Issue menu cannot popup now.

**Comment by blueworrybear** [comment]
> Issue menu cannot popup now.

Oh... I forget to attache issue menu 😓 . I'll fix it.

File: web_src/js/index.js [hunk 1]
```diff
@@ -304,11 +305,27 @@ function initImagePaste(target) {
...
+
 function initCommentForm() {
   if ($('.comment.form').length === 0) {
     return;
   }
 
+  setCommentSimpleMDE($('.comment.form textarea'));
   initBranchSelector();
   initCommentPreviewTab($('.comment.form'));
   initImagePaste($('.comment.form textarea'));
```
Comment (lunny) [inline web_src/js/index.js:331]: Isn't `initImagePaste` has done that?

Comment (blueworrybear) [reply web_src/js/index.js:331]: Sorry, I didn't understand what you mean. Do you mean why I create a new function `initSimpleMDEImagePaste` to fix the image paste problem? 🤔

**Comment by blueworrybear** [comment]
Update the `extraKeys` for simpleMDE. It's because simpleMDE does not trigger `input` event when backspace press down. It cause `tribute` fails to update it's current cursor index and fail to update menu when keywords changing.

File: web_src/js/index.js (hunk 1 above)
Comment (blueworrybear) [reply web_src/js/index.js:331]: @lunny, if you are asking why there is a new `initSimpleMDEImagePaste`, the reasons are:

1. The `initImagePaste` always attached the uploaded file input `$('.files')`, which is not the correct DOM element for comment. So I need a new function to allow caller providing correct DOM element to attach file.

2. When I trying to re-use `initImagePaste` for comment editor, it's weird that the `CodeMirror` input-field fails to insert image (`img`.  Therefore I have to insert the image markdown with `simplemde.codemirror.replaceRange`.

**Comment by lunny** [comment]
Please resolve the conflicts

**Review by zeripath — APPROVED** [review]

**Comment by blueworrybear** [comment]
> Please resolve the conflicts

No problem 😄

**Review by lunny — APPROVED** [review]

**Pull request merged**